app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -187,15 +154,15 @@ $(document).ready(function($) {
   };
 
   var add_row = function() {
-    var num_rows = $('#payout_table').find('tr').length;
+    var num_rows = $('#payout_table tbody').find('tr').length;
     var percent = num_rows <= 1 ? 100 : '';
     var denomination = $('#token_name').text();
     var amount = get_amount(percent);
     var html = `
       <tr>
         <td class="pl-0 pb-0">
           <div class="pl-0">
-            <select id="username" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
+            <select id="username" onchange="update_registry()" class="username-search custom-select" style="width: 100%; margin-left: auto; margin-right: auto;"></select>
```

Comment:
I think we should remove the Id to avoid id repetition

Reply:
@Anish-Agnihotri mind addressing that in your PR ? 😅

Reply:
For sure, just threw that into the PR as well.